LICENSE.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2008-2013, Willow Garage, Inc.
```

Comment:
shouldn't the year range and company change?

Reply:
Decent point, I'll add PickNik under Willow